tests/benchmarks/cse230/src/Week10/Axiomatic.hs

```diff
@@ -1,6 +1,5 @@
 {-@ LIQUID "--reflection"  @-}
 {-@ LIQUID "--ple"         @-}
-{-@ LIQUID "--diff"        @-}
```

Comment:
I removed the `--diff`s here to prevent outputs I didn't like and to make the timings a little more uniform.